projects/packages/forms/src/contact-form/class-contact-form-plugin.php

```diff
@@ -2046,10 +2046,7 @@ public static function parse_fields_from_content( $post_id ) {
 			if ( str_contains( $content, 'JSON_DATA' ) ) {
 				$chunks     = explode( "\nJSON_DATA", $content );
 				$all_values = json_decode( $chunks[1], true );
-				if ( is_array( $all_values ) ) {
```

Comment:
This is not related, but I observed that `$fields_array` is only used in the `else` block.